src/main/java/decodes/cwms/CwmsTimeSeriesDAO.java

```diff
@@ -154,6 +155,7 @@ public class CwmsTimeSeriesDAO
 {
 	protected static DbObjectCache<TimeSeriesIdentifier> cache = 
 		new DbObjectCache<TimeSeriesIdentifier>(60 * 60 * 1000L, false);
+	private static final long MAX_TIME_GAP_FOR_HISTORICAL = Integer.getInteger("opendcs.cwmstsdb.max_timegap_historical.days", 7) - 1L;
```

Comment:
Is this an jvm -D variable or within the (decodes|user).properties file?

generally we stick things like this within those as there's not really a good way for users to specific jvm parameter.... whether that should change though is a different question.

Reply:
That means adding it to the DecodesSettings class?

Reply:
Yeah.

Reply:
It doesn't look like DecodesSettings supports properties with names that can't be java field variables. I set it to: `cp_cwmstsdb_getNewData_max_timegap_days`

Reply:
... well that makes sense. but is a whole other update.